src/main/java/io/nats/client/support/ApiConstants.java

```diff
@@ -172,7 +172,6 @@ public interface ApiConstants {
     String RESPONSE          = "response";
     String RETENTION         = "retention";
     String SAMPLE_FREQ       = "sample_freq";
-    String SCHEMA            = "schema";
```

Comment:
This is a public variable and cannot be removed due to semver. It could be marked deprecated, but it's not hurting anything, so just remove the removal.